src/metrickit.cs

```diff
@@ -0,0 +1,356 @@
+//
+// MetricKit C# bindings
+//
+// Authors:
+//	Manuel de la Pena Saenz <mandel@microsoft.com>
+//
+// Copyright 2019 Microsoft Corporation All rights reserved.
+//
+
+using System;
+
+using CoreFoundation;
+using Foundation;
+using ObjCRuntime;
+
+namespace MetricKit {
+
+	interface NSUnitDuration : NSUnit { }
+	interface NSUnitInformationStorage : NSUnit { }
+
+	[NoWatch, NoTV, NoMac, iOS (13,0)]
```

Comment:
Should this have the catalyst annotation on it?

Reply:
Headers don't say it, but docs do.. :/

Reply:
Yeah we tend to follow headers here since it is what is is actually shipped and used.

Reply:
we'll need more tooling (both intro and xtro at least) to cover what's in catalyst and what's not - docs and headers are never enough :(